hoodie-client/src/main/java/com/uber/hoodie/index/bloom/HoodieGlobalBloomIndex.java

```diff
@@ -88,23 +85,43 @@ public HoodieGlobalBloomIndex(HoodieWriteConfig config) {
                 .map(e2 -> new Tuple2<>(e1.getKey(), e2)))
             .collect(Collectors.toList());
 
+    Set<String> filesWithNoRanges = new HashSet<>();
```

Comment:
Could you refactor the code to avoid duplicate code here (similar code is repeated above) ? My concern is we might miss adding pieces of code to both places as the classes evolve..